<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Theory</class>
 <widget class="QDialog" name="Theory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Теория</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="backBtn">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>&lt; Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insBtn">
         <property name="minimumSize">
          <size>
           <width>131</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировка вставками</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mergeBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Сортировка слиянием</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="timSortBtn">
         <property name="minimumSize">
          <size>
           <width>111</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Алгоритм Timsort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWebView" name="webView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>backBtn</sender>
   <signal>clicked()</signal>
   <receiver>Theory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
